### ISSUE TITLE
ANN mode still prints out SCATE in progress messages

### ISSUE THREAD

**Issue by AmyOlex** [AmyOlex/Chrono#84]
When I select the ANN mode the progress messages still print out "Parsing in SCATE Mode" even though it does correctly print out ANN files.

**Comment by maffeyl** [comment]
Yeah, I changed the way print mode gets tracked in lukedev, so this will be fixed once we merge.
